src/pages/post-purchase.tsx

```diff
@@ -0,0 +1,477 @@
+                      <label
+                        htmlFor="email"
+                        className="block text-sm leading-5 text-gray-800 dark:text-gray-200"
+                      >
+                        {label}
+                      </label>
+                      <div className="relative mt-1 rounded-md shadow-sm">
+                        <textarea
+                          id="response"
+                          rows={8}
+                          value={values.response}
+                          onChange={handleChange}
+                          onBlur={handleBlur}
+                          placeholder="Add you thoughts here..."
+                          className="block w-full text-black placeholder-gray-400 border-gray-300 rounded-md shadow-sm autofill:text-fill-black focus:ring-indigo-500 focus:border-blue-500 "
+                          required
+                        />
+                      </div>
+                      <div className="flex items-center justify-center w-full">
+                        <button
+                          type="submit"
+                          disabled={isSubmitting}
+                          className="px-5 py-3 mt-4 font-semibold text-white transition-all duration-150 ease-in-out bg-blue-600 rounded-md hover:bg-blue-700 active:bg-blue-800 hover:scale-105 hover:shadow-xl"
+                        >
+                          {button}
+                        </button>
+                      </div>
+                    </form>
+                  </>
+                )
+              }}
+            </Formik>
+          )}
+          {isSubmitted && (
+            <div className="space-y-2 leading-tight text-center">
+              <h3 className="text-lg font-semibold text-gray-900 leading-tighter dark:text-gray-100">
+                We've got it.
+              </h3>
+              <p className="mt-2 text-gray-900 dark:text-gray-100">
+                We use this feedback to make egghead a better learning
+                experience for everyone.
+              </p>
+            </div>
+          )}
+          {isError && (
+            <div>
+              <p>
+                Something Went Wrong{' '}
+                <span role="img" aria-label="sweating">
+                  😅
+                </span>
+              </p>
+              <p className="pt-3">
+                Are you using an aggressive ad blocker such as Privacy Badger?
+                Please disable it for this site and reload the page to try
+                again.
+              </p>
+              <p className="pt-3">
+                If you <strong>aren't</strong> running aggressive adblocking
+                please check the console for errors and email support@egghead.io
+                with any info and we will help you ASAP.
+              </p>
+            </div>
+          )}
+        </div>
+      </div>
+    </div>
+  )
+}
+
+//! delete
```

Comment:
If we were to use this, I would delete from here on and move `PostPurchaseSurvey` into the components folder and use in `NewMemberConfirmation`